src/dal_select2/views.py

```diff
@@ -21,6 +21,7 @@ def get_results(self, context):
             {
                 'id': self.get_result_value(result),
                 'text': self.get_result_label(result),
+                'selected_text': self.get_selected_result_label(result),
```

Comment:
I think it is worth adding this field only if it is different from ``text``. Now one text is sent in two fields even when it is identical.

Reply:
That's true but we're going to merge this as is so if you got hard feelings about this then perhaps make another pr or open a ticket :joy: